src/platform/forms-system/src/js/definitions/content/directDeposit.jsx

```diff
@@ -8,7 +8,7 @@ export const directDepositDescription = () => {
   return (
     <div className="vads-u-margin-top--2">
       <img
-        src="/img/direct-deposit-check-guide.png"
+        src="/img/direct-deposit-check-guide.svg"
```

Comment:
is there a separate PR open in content-build to add this image?

Reply:
Hi Tim, [here](https://github.com/department-of-veterans-affairs/content-build/pull/886) is the PR for content-build.

Reply:
We would need to merge the content build PR first, correct? Will it fail the vets-website build if we don't?

Reply:
the build won't fail, but the image will be broken. I commented on the content-build PR

Reply:
Confirmed that this image is in prod now http://content.www.va.gov.s3-website-us-gov-west-1.amazonaws.com/img/direct-deposit-check-guide.svg